tests/integration/targets/helm/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/openshift
```

Comment:
Note: For integration tests, we now have a molecule setup (https://github.com/ansible-collections/kubernetes/tree/master/molecule/default) which runs against a full Kind cluster, meaning you don't just have API access now, you can test anything against it (including Helm 2 / Tiller, I think).

Can you move all the integration tests to the molecule converge playbook (maybe group them under `tasks/helm` in a directory since there are so many)?